docs/user/simultaneous_use.rst

```diff
@@ -0,0 +1,33 @@
+Simultaneous session limits (Simultaneous-Use)
+==============================================
+
+``Simultaneous-Use`` limits how many sessions a user can have active at
```

Comment:
Here we need to explain why we had to implement this functionality in openwisp: for supporting multi-tenancy.

Please explain how to enable this feature with a short tutorial (eg: add radius group check, etc) with a screenshot.